cmd/saml2aws/commands/configure.go

```diff
@@ -5,13 +5,14 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/versent/saml2aws"
+	"github.com/versent/saml2aws/cmd/saml2aws/commands/flags"
```

Comment:
Would it be better to just move this into a package as in `github.com/versent/saml2aws/pkg/flags`?

Reply:
Right, will do - I really wasn't sure where it was meant to go, I just put in under commands since that seemed topical to its usage 😄